tests/common/runner.rs

```diff
@@ -45,35 +45,35 @@ pub fn test_execute_revm<S: Storage + Clone + Send + Sync>(storage: S, txs: Vec<
     );
 }
 
+fn encode_receipt_2718(tx_type: u8, tx_result: &PevmTxExecutionResult) -> Bytes {
+    let tx_type = TxType::try_from(tx_type).unwrap();
+    let receipt_with_bloom = tx_result.receipt.clone().with_bloom();
```

Comment:
Do we need anything else from `tx_result` for OP? If not I think passing in an owned `receipt` or `receipt_with_bloom` here would:
- Better Match the function name.
- Be tighter in case we need to encode receipts in a context without `PevmTxExecutionResult`.